restapi/src/test/java/io/stargate/web/docsapi/dao/DocumentDBTest.java

```diff
@@ -336,6 +336,11 @@ public boolean supportsSecondaryIndex() {
       return true;
     }
 
+    @Override
+    public boolean supportsSAI() {
+      return true;
```

Comment:
can we create a test that validates both SAI `true` and `false`?

Reply:
SAI `true` -> DSE integration tests, SAI 'false` -> C* 3.11 and 4.0 integration tests.

Reply:
I actually updated `DocumentApiV2TestDisable2i` (and force-pushed) because it failed :)